presto-connector-tests/src/main/java/com/facebook/presto/connector/unittest/CreatesSchemas.java

```diff
@@ -26,7 +26,7 @@
 import static java.util.Objects.requireNonNull;
 
 public interface CreatesSchemas
-        extends SPITest
+        extends TestSPI
```

Comment:
ah.... I think I mislead you. 

Typically there is a `BaseSomethingTest` and `TestSometingSpecific`. See `BaseRuleTest` and `TestSomeRule`. So here the naming IMO should be `TestSchemaCreate` and `BaseSpiTest`. Notice `Spi` not `SPI`, like `Orc` not `ORC`.